qa/integration/specs/monitoring_api_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../framework/fixture'
+require_relative '../framework/settings'
+require_relative '../services/logstash'
+
+describe "Monitoring API", :integration => true do
+  before(:all) {
+    @fixture = Fixture.new(__FILE__)
+  }
+
+  after(:all) {
+    @fixture.teardown
+  }
+
+  it "can retrieve event stats" do
+    logstash_service = @fixture.get_service("logstash")
+    logstash_service.start_with_stdin
+    5.times { logstash_service.write_to_stdin("Hello world") }
+    # TODO: get rid of this sleep, or loop
+    sleep 3
+    # check metrics
+    result = logstash_service.monitoring_api.event_stats
+    expect(result["in"]).to eq(5)
+  end
```

Comment:
If this project depends on `logstash-devutils`, it could use `try {  expect(result["in"]).to eq(5) }`, this would try multiple time with incremental sleep in between.